### DISCUSSION TITLE
Error al correr línea para entregar el TP

### DISCUSSION THREAD

**Discussion by LeoSabaLagos** [dds-jv/foro#202] [Consultas de herramientas] [answered]
Cuando corro esta linea en la terminal :
mvn clean verify && git tag entrega-final && git push origin HEAD --tags

Me tira el siguiente error
![image](https://user-images.githubusercontent.com/62613752/116313803-a030b380-a784-11eb-835a-18bb2075892d.png)

el comando git tag entrega-final no deberia crearlo al tag, o tengo que crearlo a mano desde la pagina del repo antes de ejecutar el comando?

**Comment by RafaelAlexander** [comment, marked as answer]
Debe ser porque el tag ya existe. Lo corriste anteriormente?
Si es así, hace lo siguiente:
1)Este comando es para borrar localmente:
git tag --delete entrega-final 
2)Este para borrarlo en el servidor:
git push --delete origin entrega-final 
3)Y otra vez  ejecuta el comando:
mvn clean verify && git tag entrega-final && git push origin HEAD --tags

**Reply by gastonprieto** [reply]
te sirvio @LeoSabaLagos ?

**Reply by LeoSabaLagos** [reply]
Sisi me sirvió, muchas gracias!
